addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -260,8 +260,10 @@ def _compute_number_of_hours_display(self):
         for allocation in self:
             if allocation.parent_id and allocation.parent_id.type_request_unit == "hour":
                 allocation.number_of_hours_display = allocation.number_of_days * HOURS_PER_DAY
-            else:
+            elif allocation.number_of_days:
                 allocation.number_of_hours_display = allocation.number_of_days * (allocation.employee_id.sudo().resource_id.calendar_id.hours_per_day or HOURS_PER_DAY)
+            else:
+                allocation.number_of_hours_display = allocation.employee_id.resource_calendar_id.hours_per_day or HOURS_PER_DAY
```

Comment:
They are a sudo 2 lines above coming from commit https://github.com/odoo/odoo/commit/1e8ff733d3c38e43efe528ee6bf74d78377af1fc. 
Maybe you should check if you don't need it, too.

Reply:
@jbm-odoo , right 👍 , It will require the sudo. Done the changes. Thanks